### PR TITLE
feature/47: find eslint rules

### PR DESCRIPTION
**Pull request by Danilqa**
Added:
2.6 https://eslint.org/docs/2.13.1/rules/no-extra-parens
2.7.3 https://eslint.org/docs/rules/line-comment-position
3.5.1.A https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/rules/no-useless-constructor.ts
3.6.9. https://eslint.org/docs/rules/no-param-reassign
3.9.5.c https://eslint.org/docs/2.0.0/rules/yoda
3.9.5.g https://eslint.org/docs/rules/no-else-return
3.11.1 https://eslint.org/docs/rules/no-implicit-coercion

### CODE REVIEW

**Comment by derlagg** [comment]
The name of the commits is incorrect. See this [rule ](https://github.com/eigen-space/codestyle/tree/dev/doc/common#512-%D1%81%D0%BE%D0%BE%D0%B1%D1%89%D0%B5%D0%BD%D0%B8%D0%B5-%D0%BA-%D0%B8%D0%B7%D0%BC%D0%B5%D0%BD%D0%B5%D0%BD%D0%B8%D1%8E)

**Comment by derlagg** [comment]
Please, see [plugin](https://github.com/getify/eslint-plugin-proper-ternary) about [rule 2.4.4 ](https://github.com/eigen-space/codestyle/blob/origin/feature/47-find-eslint-rules/doc/scripts/README.ru.md#244-%D0%BD%D0%B5-%D0%B0%D0%B2%D1%82%D0%BE%D0%BC%D0%B0%D1%82%D0%B8%D0%B7%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%BE-%D1%82%D0%B5%D1%80%D0%BD%D0%B0%D1%80%D0%BD%D1%8B%D0%B9-%D0%BE%D0%BF%D0%B5%D1%80%D0%B0%D1%82%D0%BE%D1%80)

File: doc/scripts/README.md [hunk 1]
```diff
@@ -932,7 +932,7 @@ b. Внутри литерала массива после `[` и перед `]`
    }
```
Comment (derlagg) [inline doc/scripts/README.md]: These changes will be relevant for the version README.ru.md. We need not to get confused and have all the current changes in only one file.

Comment (Danilqa) [reply doc/scripts/README.md]: Done)

File: doc/scripts/README.md [hunk 2]
```diff
@@ -1614,9 +1614,9 @@ d. После spread-оператора не должно быть пробел
 
 ### 3.5. Классы
 
-#### 3.5.1. \[Не автоматизировано\] Конструктор
+#### 3.5.1. \[Частично втоматизировано\] Конструктор
 
-a. \[Не автоматизировано. Не работает no-useless-constructor\] Не нужно объявлять конструктор, если он не используется.
+a. \[Автоматизировано: no-useless-constructor\] Не нужно объявлять конструктор, если он не используется.
 
 b. \[Автоматизировано: tslint\] Для подклассов обязательно вызывать `super()` в самом начале конструктора,
```
[the remark this answers is not in the document]
Comment (Danilqa) [reply doc/scripts/README.md]: I've installed it

File: doc/scripts/README.md [hunk 3]
```diff
@@ -2054,8 +2054,9 @@ d. Не использовать выход из функции, если при
     });
 ```
 
-#### 3.6.9. \[Не автоматизировано\] Возвращаемое значение
+#### 3.6.9. \[Автоматизировано\] Возвращаемое значение
 
+[Автоматизировано: no-param-reassign] \
 a. Возвращать значение из функции с помощью `return`, а не через параметры функции.
```
[the remark this answers is not in the document]
Comment (Danilqa) [reply doc/scripts/README.md]: Fixed

File: doc/scripts/README.md [hunk 4]
```diff
@@ -2828,7 +2846,7 @@ i. \[Не автоматизировано. Не работает no-nested-tern
```
[the remark this answers is not in the document]
Comment (Danilqa) [reply doc/scripts/README.md]: Done

**Pull request merged**